Tasks/AzureFileCopy/AzureFileCopyJob.ps1

```diff
@@ -52,7 +52,15 @@ param (
 
     Write-Verbose "Initiating copy on $fqdn " -Verbose
 
-    [String]$copyToAzureMachinesBlockString = "Copy-ToAzureMachines -MachineDnsName `$fqdn -StorageAccountName `$storageAccount -ContainerName `$containerName -SasToken `$sasToken -DestinationPath `$targetPath -Credential `$credential -AzCopyLocation `$azCopyLocation -AdditionalArguments `$additionalArguments -WinRMPort $winRMPort $cleanTargetPathOption $skipCACheckOption $httpProtocolOption $enableDetailedLoggingOption"
+    [String]$copyToAzureMachinesBlockString = [string]::Empty
+    if([string]::IsNullOrWhiteSpace($additionalArguments))
```

Comment:
Might be ok for now. Should we have a stringbuilder equivalent  to build the command? In future, if there is another parameter introduced, the if else blocks can become too complicated.